packages/docusaurus-theme-classic/src/theme/BlogListPage/index.js

```diff
@@ -7,15 +7,21 @@
 
 import React from 'react';
 
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
 import BlogPostItem from '@theme/BlogPostItem';
 import BlogListPaginator from '@theme/BlogListPaginator';
 
 function BlogListPage(props) {
   const {metadata, items} = props;
+  const {
+    siteConfig: {title: siteTitle},
+  } = useDocusaurusContext();
+  const isBlogOnlyMode = metadata.permalink === '/';
+  const title = isBlogOnlyMode ? siteTitle : 'Blog';
 
   return (
-    <Layout title="Blog" description="Blog">
+    <Layout title={title} description="Blog">
```

Comment:
> description="Blog"

By the way, I’m not sure that it’s a good idea to specify a description for this page, since this is optional, and in its current form it is too short.

Reply:
Would it be better as `Blog | ${title}`?

Reply:
I don’t think so, because it was assumed that the title would be completely its own without adding anything else.